bin/inx-chronicle/src/api/error.rs

```diff
@@ -12,41 +12,59 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 #[allow(missing_docs)]
-pub enum APIError {
-    #[error("No results returned!")]
+pub enum InternalApiError {
+    #[error(transparent)]
+    Hyper(#[from] hyper::Error),
+    #[error(transparent)]
+    MongoDb(#[from] mongodb::error::Error),
+    #[error(transparent)]
+    ValueAccess(#[from] ValueAccessError),
+    #[error(transparent)]
+    Doc(#[from] DocError),
+    #[error(transparent)]
+    UnexpectedLedgerInclusionState(#[from] UnexpectedLedgerInclusionState),
+    #[cfg(feature = "stardust")]
+    #[error(transparent)]
+    Stardust(#[from] chronicle::stardust::Error),
+    #[error(transparent)]
+    UrlEncoding(#[from] serde_urlencoded::de::Error),
+}
```

Comment:
❤️

Reply:
Surprised you didn't tell me to order it alphabetically

Reply:
Which you already did 😊 — thanks!